suite-native/alerts/src/components/AlertSheet.tsx

```diff
@@ -87,12 +94,16 @@ export const AlertSheet = ({ alert }: AlertSheetProps) => {
                     >
                         <Card style={applyStyle(alertSheetContainerStyle)}>
                             <VStack style={applyStyle(alertSheetContentStyle)} spacing="large">
-                                <Pictogram
-                                    title={title}
-                                    variant={pictogramVariant}
-                                    subtitle={description}
-                                    icon={icon}
-                                />
+                                {icon && pictogramVariant ? (
+                                    <PictogramTitleHeader
+                                        title={title}
+                                        variant={pictogramVariant}
+                                        subtitle={description}
+                                        icon={icon}
+                                    />
+                                ) : (
+                                    <CenteredTitleHeader title={title} subtitle={description} />
+                                )}
```

Comment:
This is total nitpick, feel free to read and then ignore (resolve) right away, but since we were touching this, here is the way I would personally use the components composition here:
```suggestion
                                <VStack alignItems="center" spacing="large">
                                  { icon && pictogramVariant && (<Pictogram variant={variant} icon={icon} size={size} />) }
                                  <CenteredTitleHeader title={title} subtitle={description} />
                                </VStack>
```

Reply:
76f750ecc5